src/pages/components/tags.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Tags
+description: >
+  A Tag is an interactive label used to categorize content.
+package: '@zendeskgarden/react-tags'
+components:
+  - tags/src/elements/Tag.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To categorize by dimension, attribute, or property</li>
+      <li>To represent filter parameters</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>To prompt an action, use a [Button](/components/button) instead</Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Tag component.
+
+import TagsDefault from '../../examples/tags/TagsDefault.tsx';
+import TagsDefaultCode from '!!raw-loader!../../examples/tags/TagsDefault.tsx';
+
+<CodeExample code={TagsDefaultCode}>
+  <TagsDefault />
+</CodeExample>
+
+### Avatar
+
+Medium and large Tags can contain an Avatar.
+
+import TagsAvatar from '../../examples/tags/TagsAvatar.tsx';
+import TagsAvatarCode from '!!raw-loader!../../examples/tags/TagsAvatar.tsx';
+
+<CodeExample code={TagsAvatarCode}>
+  <TagsAvatar />
+</CodeExample>
+
+### Close
+
+Tags can be dismissed.
+
+import TagsClose from '../../examples/tags/TagsClose.tsx';
+import TagsCloseCode from '!!raw-loader!../../examples/tags/TagsClose.tsx';
+
+<CodeExample code={TagsCloseCode}>
+  <TagsClose />
+</CodeExample>
+
+### Color
+
+Tags can be colored using a [theme
+palette](https://zendeskgarden.github.io/react-components/theming/#palette) hue
+or hex value.
+
+import TagsColor from '../../examples/tags/TagsColor.tsx';
+import TagsColorCode from '!!raw-loader!../../examples/tags/TagsColor.tsx';
+
+<CodeExample code={TagsColorCode}>
+  <TagsColor />
+</CodeExample>
+
+### Shape
+
+Tags are rectangular by default, but they can be pill or round too. Round tags
```

Comment:
Tags are rectangular by default. Other options are pill and round. Round tags are reserved for very short content like two digit numbers.